.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "recommendations": [
```

Comment:
We can consider adding https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint as there is even a GitHub workflow which uses `markdownlint-cli` to check the markdown files.

Reply:
Added.